### DISCUSSION TITLE
Is it possible to reposition the group window pop up?

### DISCUSSION THREAD

**Discussion by winteriscariot** [LBCrion/sfwbar#213] [Q&A]
![1](https://github.com/user-attachments/assets/944ebd38-fe2e-4000-b5f6-1d3d47107d01)

I'd like to anchor these pop ups in the middle of the firefox icon, is that possible via css?

**Comment by LBCrion** [comment]
Sure. I added properties `-GtkWidget-widget-anchor` and `-GtkWidget-window-anchor` to the latest git version. These allow you to explicitly specify how a popup window will be positioned vs the parent widget. The possible values are `northwest`, `north`, `northeast`, `east`, `center`, `west`, `southwest`, `south`, `southeast`, `static` and `default`. 
The properties specify a specific edge or a corner of the widget that will be aligned with a specific edge or corner or a window. I.e. to have a center alignment of a window above widget you want to specify `-GtkWidget-widget-anchor: north` and `-GtkWidget-window-anchor: south`. So the southern (bottom) edge of the window is attached to the northern (top) edge of a widget.

**Comment by ffs-akl** [comment]
@winteriscariot  Would you mind sharing your config file of sfwbar ?
